.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: trusty
+language: csharp
+solution: Banshee.sln
+
+addons:
+  apt:
+    packages:
+      - libgtk3.0-cil
+      - libdbus2.0-cil
+      - libtaglib2.1-cil
+      - libmono-addins0.2-cil
+      - libdbus-glib2.0-cil
+      - libgudev3.0-cil
+      - libmono-zeroconf1.0-cil
+      - libgconf2.0-cil
+env:
+  - BUILD_CONFIGURATION=Release
+  - BUILD_CONFIGURATION=Debug
+
+script:
+  - msbuild /p:Configuration=$BUILD_CONFIGURATION Banshee.sln
```

Comment:
I'd like to build it both ways (Solution and autotools) to ensure compatibility between both build systems. Especially since you can't really use autotools on Windows.

This PR should be just the first step. And since scripting the msbuild part was easier, I did that for starters ;)